WordPress/build.gradle

```diff
@@ -13,6 +13,57 @@ plugins {
     id "com.google.dagger.hilt.android"
     id "org.jetbrains.kotlinx.kover"
     id "com.google.devtools.ksp"
+    id "com.osacky.fladle"
+}
+
+fladle {
+    testTargets = [
+            "notPackage org.wordpress.android.ui.screenshots"
+    ]
+    configs {
+        wordpress {
+            def app = "wordpress"
+            debugApk.set(pathForVariant(app, "debugApk"))
+            instrumentationApk.set(pathForVariant(app, "instrumentationApk"))
+            smartFlankGcsPath.set(pathForSmartFlankGcsPath(app))
+            testTargets.set(testTargets.get() + [
+                    "notClass org.wordpress.android.e2e.StatsTests",
+                    "notClass org.wordpress.android.e2e.StatsGranularTabsTest",
+            ]
+            )
+        }
+        jetpack {
+            def app = "jetpack"
+            debugApk.set(pathForVariant(app, "debugApk"))
+            instrumentationApk.set(pathForVariant(app, "instrumentationApk"))
+            smartFlankGcsPath.set(pathForSmartFlankGcsPath(app))
+        }
+    }
+    serviceAccountCredentials = new File(System.getProperty("user.home"), ".configure/wordpress-android/secrets/firebase.secrets.json")
```

Comment:
Question (❓): You are using `user.home` + `.configure` instead of `project.home` + `.configure-files`. Let's apply this change on WCAndroid too, wdyt? 🤔

Reply:
It's specification of projects, WC and WP/JP has this file's destination configured differently: 
- [WooCommerce](https://github.com/woocommerce/woocommerce-android/blob/trunk/.configure#L43)
- [WordPress/Jetpack](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/.configure#L48)

Reply:
Ok, cool, so since both those projects have a different `destination`, we should use that and adhere to in on the `fladle` side too, gotcha, I was just trying to make both project consistent, but this seems a bit more work than I expected... 🫣